testdata/test.git/config

```diff
@@ -34,3 +34,6 @@
 [remote "garbage"]
 	url = garbageurl
 	fetch = +refs/heads/*:refs/remotes/garbage/*
+[remote "origin-subfolder"]
+	url = git://gitlab.com/zaquestion/sub/folder/test.git
```

Comment:
If you're up for it I'd appreciate a 2nd test in the scp-style format. We'll call that non-blocking tho